community/modules/scripts/ramble-setup/README.md

```diff
@@ -0,0 +1,83 @@
+## Description
+
+This module will create a set of startup-script runners that will setup Ramble,
+and install Ramble’s dependencies.
+
+For more information about ramble, see: https://github.com/GoogleCloudPlatform/ramble
+
+This module outputs two startup script runners, which can be added to startup
+scripts to setup, ramble and its dependencies.
+
+For this module to be completely functionaly, it depends on a spack
+installation. For more information, see HPC-Toolkit’s Spack module.
+
+> **_NOTE:_** This is an experimental module and the functionality and
+> documentation will likely be updated in the near future. This module has only
+> been tested in limited capacity.
+
+# Examples
+
+## Basic Example
+
+```yaml
+- id: ramble-setup
+  source: community/modules/scripts/ramble-setup
```

Comment:
I think we will want some changes/additions to the examples, but we should first resolve the discussion about outputs.

Reply:
I added a more complete example. Let me know if that looks better or if you have other suggestions for examples.

Reply:
It still feels incomplete to me without showing how it is used. I can think of several ways to resolve this but my prefered option would be:
- Create a ramble.yaml example in community examples. 
- For now it could just be this module + startup script + vm-instance, showing how to use install
- As you add functionality you can grow the example to include the updates, such as additional modules
- Then in this doc you can put a not under this snipit saying "Also see full example" (similar to [gke](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/develop/community/modules/compute/gke-node-pool/README.md#example)

I would suggest starting this example in a subsequent PR so we can get this one merged. 
Do you agree. I think it will also help me to review the changes to see how they affect the example.

Reply:
I agree. I put it in this PR anyway, so I can just update it when I make the follow on PRs.

Reply:
There are a few comments on the example. You can either address the comments in this PR or move to another PR and close the comments. I don't mind either way.

Reply:
I think I addressed all of them. Thanks for pointing out all of the necessary cross references.